motor/discovery/gcp/resolver_project.go

```diff
@@ -73,6 +73,7 @@ func (r *GcpProjectResolver) Resolve(ctx context.Context, tc *providers.Config,
 
 	var resolvedRoot *asset.Asset
 	if tc.IncludesOneOfDiscoveryTarget(common.DiscoveryAuto, common.DiscoveryAll, DiscoveryProjects) {
+		pf.Name = "gcp-project"
```

Comment:
This should really not be done here

Reply:
The plan was to fix it after all assets are migrated (basically when a gcp asset no longer exists). Currently, it is still returned for a gcp org. Once that is fixed I can refactor the platform code